### ISSUE TITLE
Necrosis (DK talent) bugged

### ISSUE THREAD

**Issue by knasen88** [SunwellTracker/issues#340]
Decription: 
The talent necrosis seems to to half it's intended damage, I'm guessing it doesn't work with dual wield? have not tested 2h

How it works: 
200k melee dmg
23.5k necrosis dmg ( 4/5 black ice talent)

How it should work: 
Should be around 44k necrosis damage
200k * 0.2 * 1.08

Source (you should point out proofs of your report, please give us some source): 
it says so right there in the talent
https://i.imgur.com/hvsYMdU.png

**Comment by Vidia2** [comment]
can confirm it, saw a friend playing with exact same numbers, Necrosis doesnt give the 20% at 5 points.

**Comment by Cwiercpasiec** [comment]
So I tested it with 2 hand weapon, without other gear, with gear, with dual wield, in Unholy Presence, Blood Presence, with and without Black Ice and it always hit for 20% (rounded down) of melee attack damage (damage of Necrosis wasn't increased by Blood Presence, nor by Black Ice, and I don't think it should, but I'm not 100% sure, some kind of proof would be useful here).

I tried it on Feronis and PTR, here are some screenshots of Combat Log from PTR:
https://imgur.com/a/yYBPl 
Simply take the melee damage of an attack, multiply it by 0,2, round it down and compare it to damage of Necrosis coming right after this attack, it's always equal.

And here are logs from one of my guild's ToGC 25:
http://www.worldoflogs.com/reports/5oguxwbmxga69jf0/details/79/?s=1845&e=1999
This link leads to Lord Jaraxxus encounter details for Afganek, our frost DK, specced sub unh with Necrosis 5/5. As you can see, he did 387568 damage with melee attacks and 75670 damage with Necrosis. It's a bit less than 77513 (387568 * 0,2), but that's because of partial resists and rounding down every single attack.

In conclusion, everything seems to work well, unless Necrosis should benefit from talents like Black Ice or skills like Blood Presence directly, then the talent would be bugged, nevertheless not in the way knasen88 mentioned.

**Comment by Vraeldk** [comment]
Necrosis doesn't benefit from dmg boosts like blood presence, black ice, curse of elements etc. It only deals <20% of dk's melee attacks. In addition, necrosis has partial resist. One thing that should be fixed (really minor bug) that if melee hits for example 1344, necrosis should 269 (0,2 * 1344 = 268,8). 
Proof: 1s hit in global logs:
[17:45:18.107] Mca hits Deathbringer Saurfang 1433
[17:45:18.979] Mca Necrosis Deathbringer Saurfang 287
1433*0,2=286,6


These are global logs from wotlk: http://worldoflogs.com/reports/3m901oh238fv941t/log/?s=2276&e=2477
[17:45:18.107] Mca hits Deathbringer Saurfang 1433
[17:45:18.729] Mca crits Deathbringer Saurfang *4781*
[17:45:18.979] Mca Necrosis Deathbringer Saurfang 287
[17:45:19.290] Mca hits Deathbringer Saurfang 966
[17:45:19.758] Mca hits Deathbringer Saurfang 1634
[17:45:19.836] Mca Necrosis Deathbringer Saurfang 956
[17:45:20.212] Mca Necrosis Deathbringer Saurfang 193
[17:45:20.589] Mca Necrosis Deathbringer Saurfang 327

[17:45:23.442] Mca Necrosis Deathbringer Saurfang 312 (R: 35)
[17:45:23.814] Mca Necrosis Deathbringer Saurfang 1172
[17:45:24.610] Mca Necrosis Deathbringer Saurfang 302
[17:45:25.040] Mca Necrosis Deathbringer Saurfang 397 (R: 44)
[17:45:25.804] Mca Necrosis Deathbringer Saurfang 672 (R: 168)
[17:45:26.248] Mca Necrosis Deathbringer Saurfang 355 (R: 89)
[17:45:27.402] Mca Necrosis Deathbringer Saurfang 1192 (R: 132)
[17:45:27.826] Mca Necrosis Deathbringer Saurfang 407
[17:45:29.044] Mca Necrosis Deathbringer Saurfang 756 (R: 84)
[17:45:29.417] Mca Necrosis Deathbringer Saurfang 1258
[17:45:30.258] Mca Necrosis Deathbringer Saurfang 273 (R: 30)

This is from our realm feronis: http://www.worldoflogs.com/reports/yyc2dn1otxg5cmwk/log/?s=1701&e=1855

[20:29:33.328] Vraelxgodx crits Eydis Darkbane *6122*
[20:29:33.328] Vraelxgodx Necrosis Eydis Darkbane 1224
[20:29:33.471] Vraelxgodx hits Eydis Darkbane 1344
[20:29:33.471] Vraelxgodx Necrosis Eydis Darkbane 268
[20:29:34.862] Vraelxgodx crits Eydis Darkbane *4546*
[20:29:34.862] Vraelxgodx Necrosis Eydis Darkbane 909
[20:29:35.906] Vraelxgodx crits Eydis Darkbane *7076*
[20:29:35.906] Vraelxgodx Necrosis Eydis Darkbane 1274 (R: 141)
[20:29:36.142] Vraelxgodx crits Eydis Darkbane *4499*
[20:29:36.142] Vraelxgodx Necrosis Eydis Darkbane 899
